app/assets/stylesheets/content_block_manager/components/_textarea-component.scss

```diff
@@ -0,0 +1,6 @@
+.app-c-content-block-manager-textarea-component {
+  .guidance.govspeak-supported {
+    margin-top: 1rem;
+    margin-bottom: 1.5rem;
```

Comment:
I think the top margin is a smidge too high here. Might be worth using the `govuk-spacing` helpers too, for consistency.